### DISCUSSION TITLE
Bounding Box dilema

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#3904] [Q&A]
How is measured Bounding box in an assembly? Each part(material) in model has its own Bounding box? Or its one bounding box for whole model? 

IDEA: I would be able to change the position of annotation text according change of bounding box in an animation.... or not?

Thanks
Petr

**Comment by elalish** [comment]
The bounding box is the entire scene, at its resting pose. We don't yet support model-attached annotations, though it's on the road-map. Currently, you'd need to update their world-space position yourself.

**Comment by tyler-mccarthy** [comment]
Hi, can I ask whether the model-attached annotations have been implemented yet? I am attempting to align a partially transparent three.js layer on top of my model-viewer layer. If I was able to pass the exact position of the model viewer layer rather than the bounding box this would enable me to position objects on the three.js layer much easier.

Thanks,
Tyler McCarthy, email: tyler.mccarthy@bmwgroup.com

**Reply by elalish** [reply]
Indeed, model-attached annotations have been implemented now: https://modelviewer.dev/examples/annotations/index.html#animatedHotspots
